react/react1/week3/todo-list/src/components/Timer.js

```diff
@@ -0,0 +1,9 @@
+import { useState, useEffect } from "react";
+const Timer = () => {
+  const [count, setCount] = useState(0);
+  useEffect(() => {
+    setTimeout(() => setCount(count + 1), 1000);
+  }, [count]);
```

Comment:
This timer counts, 1+2+3+4+5+6... etc seconds, But we need the counter to count 1+1+1+1+1... etc seconds.